packages/docs/examples/va-slider/Api.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <api-docs :apiOptions="apiConfig" :componentOptions="VaSlider"></api-docs>
+</template>
+
+<script>
```

Comment:
webstorm has live templates and file templates.
I suggest you add one for vuestic. Here's the one I use

tstmp TypeScript vue template
![image](https://user-images.githubusercontent.com/5394573/81306782-8c2e0680-9088-11ea-9dfe-36702a156713.png)


```vue
<template>
  <div class="$FILE_NAME$">
    $END$
  </div>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator"

@Component({})
export default class $FILE_NAME$ extends Vue {
  
}
</script>

<style lang="scss">
.$FILE_NAME$ {
  
}
</style>
```